_posts/2018-11-13-deploy-rails.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: blog 
+title: Deploying a monolithic rails application
+author: Ngari Ndung'u
+blog-image: 
+intro: So, you finally have your app in a state that you aren't too embarrassed to show off. How do you open it up to the world?
+  How do you manage the influx of users? How will you keep it safe from the *dangerous* internets? These are questions we face every time we work on a new product.
+  This post explores one of the ways to deploy your shiny new rails application and the common tooling behind it.
+---
+
+{{ page.intro }}
+
+## Monolithic?
+
```

Comment:
Mention somewhere that it is good practice not to wait till you application is done to deploy it, but to start with a deployed boilerplate, in rails case, deploying a rails new app, and making user the changes are also always deployed.. Also mention the need for staging, production and dev environments.